### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Ndaboom**
**Math Magicians: setup**
- React repo initialized
- Linters setup
- MIT license added
- Descriptive readme.md added

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (HAMMAS-SALEEM) [inline public/manifest.json:1]: ### Your Pull Request Summary
not descriptive

### Descriptive Pull Request Summary
Descriptive Summary Initialize Math Magician Application

- [ ] You can see that your summary is not descriptive enough. Kindly update your summary and make it descriptive to make your pull request professional 😊

**Hint: Kindly update it by click on the three dots on the upper left corner of this summary box  👍**

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submisson 🔴 🚥

## Hey @Ndaboom :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Following Linters
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (HAMMAS-SALEEM) [inline package.json:1]: Capture

- [x] Your license file is not descriptive enough. Kindly add your name, current year and app type 😊 to make it descriptive 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Ndaboom :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Linters
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Ndaboom , 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights

- Linter checks are passing
- Good commit messages
- Requested changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**